### PR TITLE
Update torch-xpu-ops commit pin

### PR DESCRIPTION
**Pull request by EikanWang**
Update the torch-xpu-ops commit to [28cfac20ec662abdb0ac98faf122450013e8f520](https://github.com/intel/torch-xpu-ops/commit/28cfac20ec662abdb0ac98faf122450013e8f520), includes:

- Disable batch_norm vectorization path to fix accuracy issues.
- Fix the LSRM/RNN implementation error.

Pull Request resolved: https://github.com/pytorch/pytorch/pull/143984
Approved by: https://github.com/EikanWang, https://github.com/ruidazeng, https://github.com/desertfire, https://github.com/jansel

(cherry picked from commit 1e881ceecfe80532206ca4e0acb64391fab8b935)

cc @voznesenskym @penguinwu @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @ColinPeppler @amjames @desertfire @chauhang @aakhundov

### CODE REVIEW

**Review by atalman — APPROVED** [review]
lgtm

**Review by kit1980 — APPROVED** [review]
